zebedee-cms/src/main/java/com/github/onsdigital/zebedee/configuration/Configuration.java

```diff
@@ -59,6 +59,12 @@ public class Configuration {
     // how many additional seconds after the publish
     private static final int DEFAULT_SECONDS_TO_CACHE_AFTER_SCHEDULED_PUBLISH = 30;
 
+    private static final String RESUMABLE_TYPE = "text/plain";
+    private static final String IS_PUBLISHABLE = "true";
+    private static final String LICENCE = "Open Government Licence v3.0";
+    private static final String LICENCE_URL = "https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/";
+    private static final String UPLOAD_SERVICE_API_URL = "http://dp-upload-service:25100";
```

Comment:
This default host value should be set as localhost rather than dp-upload-service e.g. http://localhost:25100.  The specific container name should only be set in the docker-compose file environment variable for this value,

Reply:
This is now addressed.